Curs04/src/views/index.html

```diff
@@ -4,18 +4,24 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <link rel="stylesheet" href="img/">
+
+    <link rel="stylesheet/less" href="index.less" type="text/css"/>
```

Comment:
there should be only one less file imported... use the less @import instruction

http://lesscss.org/features/#features-overview-feature-importing#features-overview-feature-importing

Reply:
Where should I put the @import instruction?

In html file won't work..

Reply:
in the main less file, check the link I left on the previous comment... those are all less instructions